### PR TITLE
[SPARK-270116] Add RenderConfig for Input Elements (Windows)

### PR DESCRIPTION
**Pull request by dgharde**
## Related Issue
Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
the issues fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one if reasonable to aid 
in issue tracking.

**NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
page above.

## Description

Add a new RenderConfig class which can be used for all elements

## Sample Card
If appropriate, please include a link to a card in one of the samples directories that can be used to validate this change. This can be an existing card or a card added with this PR.

## How Verified
How you verified the fix, including one or all of the following: 
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it 
will aid in code reviews or corresponding fixes on other platforms for eg.***

### CODE REVIEW

**Review by ggovardh — APPROVED** [review]

**Review by rj100189 — CHANGES_REQUESTED** [review]
To be updated as per discussion.

File: source/qml/Library/RendererQml/AdaptiveCardRenderConfig.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        //Sample Values
+        std::string multiLineTextHeight{ "20" };
+    };
+
+    //Holds references to all elements
+    class AdaptiveCardRenderConfig
+    {
+    public:
+        AdaptiveCardRenderConfig(bool isDarkMode = true);
+        const bool isDarkMode();
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardRenderConfig.h:28]:
```suggestion
        bool isDarkMode() const;
```

File: source/qml/Library/RendererQml/AdaptiveCardRenderConfig.cpp [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#include "AdaptiveCardRenderConfig.h"
+
+namespace RendererQml
+{
+   
+    AdaptiveCardRenderConfig::AdaptiveCardRenderConfig(bool isDarkMode)
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardRenderConfig.cpp:6]:
```suggestion
    AdaptiveCardRenderConfig::AdaptiveCardRenderConfig(bool isDarkMode)
    : isDark(isDarkMode)
```

File: source/qml/Library/RendererQml/AdaptiveCardRenderConfig.h [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        InputTextConfig getInputTextConfig();
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardRenderConfig.h:29]:
```suggestion
        InputTextConfig getInputTextConfig() const;
```

File: source/qml/Library/RendererQml/AdaptiveCardRenderConfig.h [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        void setInputTextConfig(InputTextConfig config);
+
+    private:
+        bool isDark;
+        InputTextConfig textInputConfig;
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardRenderConfig.h:34]:
```suggestion
        bool m_isDark;
        InputTextConfig m_textInputConfig;
```

**Review by rj100189 — APPROVED** [review]

**Pull request merged**